tools/deployment/package-helm/templates/query-worker-deployment.yaml

```
@@ -0,0 +1,100 @@
+              value: "/opt/clp/lib/python3/site-packages"
+          volumeMounts:
+            - name: "config"
+              mountPath: "/etc/clp-config.yaml"
+              subPath: "clp-config.yaml"
+              readOnly: true
+            - name: {{ include "clp.volumeName" (dict
+                "component_category" "query-worker"
+                "name" "logs"
+              ) | quote }}
+              mountPath: "/var/log/query_worker"
+            - name: {{ include "clp.volumeName" (dict
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)